### ISSUE TITLE
Tabs are not counted as 4 spaces

### ISSUE THREAD

**Issue by JulienCochuyt** [jleclanche/tan#3]
Howdy! Sorry you're having trouble. To expedite your experience,
provide some basics for me:

Operating system: Windows 10 64 bits
Python version: 3.6.3
*tan* version: 19.1.1
Does also happen on master: Yes

First of all: I need to indent with tabs. I need a code formatter. Thank you.

Reading your description in ambv#513, I understand you intended to strictly count 1 tab as 4 spaces.

If so, please consider the following code, formatted with `tan --use-tabs`:
```python
{
    "8_______": {
        "8_______": {
            "8_______": {"8_______": {"8_______": {"8_______": "12__________"}}}
        }
    }
}
# Above: Indented with 4-spaces, reaches column 80, left unchanged due to #2.

# Below: Indented with tabs, reaches column 90 if counting 1 tab = 4 spaces
{
	"8_______": {
		"8_______": {"8_______": {"8_______": {"8_______": {"8_______": "12__________"}}}}
	}
}
```

**Comment by jleclanche** [comment]
I think this is fixed with tan 20.0.2. I rebased on top of latest black, which I believe fixed the underlying issue.

**Comment by JulienCochuyt** [comment]
> I think this is fixed with tan 20.0.2. I rebased on top of latest black, which I believe fixed the underlying issue.

Nope.
I just ran the same test with latest master and obtained the same faulty results.
I doubt this is ever going to be fixed on black's side, as they seem to refuse considering tabs as an option.
I've finally found a fix for this by amending `is_line_short_enough` so that it properly counts even if a `line_str` is passed in.
I'll try to file this fix as a PR shortly.
